config/config.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+type Config struct {
+	GinMode       string `env:"GIN_MODE" envDefault:"debug"`
+	AdminPassword string `env:"ADMIN_PASSWORD" envDefault:"xxx"`
```

Comment:
"Needs to be at least 10 chars long otherwise the admin page will not be available" ?